tools/c7n_azure/c7n_azure/function_package.py

```diff
@@ -120,23 +121,35 @@ def _update_perms_package(self):
 
     @property
     def cache_folder(self):
+        if self.cache_override_path:
+            return self.cache_override_path
+
         c7n_azure_root = os.path.dirname(__file__)
         return os.path.join(c7n_azure_root, 'cache')
 
-    def build(self, policy, modules, non_binary_packages, excluded_packages, queue_name=None,):
+    def build(self, policy, modules, non_binary_packages, excluded_packages, queue_name=None):
+        cache_zip_file = self.build_cache(modules, excluded_packages, non_binary_packages)
+
+        self.pkg = PythonPackageArchive(cache_file=cache_zip_file)
 
+        exclude = os.path.normpath('/cache/') + os.path.sep
```

Comment:
nit: I think it needs to be `self.cache_folder` instead of `/cache/` now (or we even don't need it because there is no `.py` files in the cache folder now)